docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.9"
+
+services:
+  syncmymoodle:
+    build: .
+    restart: always
+    volumes:
+      # change the following lines
```

Comment:
Alternatively we could add `docker-compose.local.yaml` to `.gitignore` and tell users to overwrite the directories there (passing both via the `-f` flag to docker compose. This way git does not always complain about unstaged changes.

Reply:
My intuition was: rename the repo to `docker-compose.yaml.template` or `example` and make users copy to and customize it in `docker-compose.yaml` which is `.gitignore`d.

Reply:
I guess both should work equally well. Most of the time I see the approach I proposed as it allow starting without any copying etc and allows you to change just a single setting while having the rest profit from updates etc however I guess this comes down to personal preference and I am fine either way (as long as one can make adjustments without tainting the repo)

Reply:
Your reasoning is solid. As far as I can see the question is about forcing user agency i.e. whether it would run as-is but be modifiable vs it wouldn't run and you *need* to change the setting for it to run?